### PR TITLE
auto append https://

### LINKED ISSUE #1: Doesn't prepend HTTPS://
I like this tool a lot, but I keep getting `Invalid URL 'demo.station70.com': No scheme supplied. Perhaps you meant https://demo.station70.com?` errors. Maybe it could automatically prepend `https://` when I forgot to do that?

### PR DESCRIPTION
**Pull request by the-real-finnventor**
fixes #1

### CODE REVIEW

File: get.py [hunk 1]
```diff
@@ -13,13 +13,16 @@
...
-    output = requests.get(args.url).text
-except requests.exceptions.MissingSchema:
-    exit(
-        f"Invalid URL '{args.url}': No scheme supplied. Perhaps you meant https://{args.url}?")
-except requests.exceptions.ConnectionError:
-    exit(f"404: URL not found")
+
+def set_output():
+    try:
+        return requests.get(args.url).text
```
Comment (ace0) [inline get.py]: The other way to do this instead of except handling is to `beginswith()` and then add it if that fails.

File: get.py [hunk 2]
```diff
@@ -13,13 +13,16 @@
...
+    except requests.exceptions.MissingSchema:
+        args.url = f"https://{args.url}"
+        return set_output()
```
Comment (ace0) [inline get.py]: Doing it recursively this way could lead to an infinite loop.

**Comment by the-real-finnventor** [comment]
fixes #4

**Review by ace0 — APPROVED** [review]
lgtm

**Pull request merged**